src/app/js/formats/sparql/SparqlTextField/SparqlView.js

```diff
@@ -53,9 +54,42 @@ const styles = {
         color: 'rgb(158, 158, 158)',
         fontSize: '1.3rem',
     },
+    array: {
+        flexGrow: '2',
+        fontSize: '1.5rem',
+        textDecoration: 'none',
+        margin: 0,
+    },
 };
 
 export class SparqlTextField extends Component {
+    ifArray = result => {
+        const { className, sparql } = this.props;
+        const temp = clonedeep(result);
+        if (temp[1].value.includes(sparql.separator)) {
```

Comment:
Aren't there any cases where `temp[1]` is undefined?
And thus where `temp[1].value` throws an error (I didn't succeed in triggering such a case, but I don't know the structure of `obj` from `topairs`).

Reply:
toPairs transform a json objet in array : 
attribute => [0] (the label)
value => [1] (a json object) 
![image](https://user-images.githubusercontent.com/31852366/41332144-f1cc1470-6edc-11e8-9fbd-7455a76d5485.png)